torch/_export/__init__.py

```diff
@@ -674,6 +762,52 @@ def _export(
     constraints = constraints or []
     kwargs = kwargs or {}
 
+    if not strict:
+        assert isinstance(f, torch.nn.Module)
+        assert len(preserve_module_call_signature) == 0
+        assert len(constraints) == 0, "dynamic shape NYI"
+        assert len(kwargs) == 0, "keyword arguments NYI"
+        out_spec = None
+
+        def _tuplify_outputs(aot_export):
```

Comment:
Can you leave a comment here explaining we don't actually use aot_export_module's out_spec so it is up to us to manipulate it however we want?